include/SFML/System/Angle.hpp

```diff
@@ -0,0 +1,426 @@
+[[nodiscard]] constexpr Angle operator /(Angle left, float right);
+
+////////////////////////////////////////////////////////////
+/// \relates Angle
+/// \brief Overload of binary / operator to scale an angle value
+///
+/// \param left  Left operand (an angle)
+/// \param right Right operand (a number)
+///
+/// \return \a left divided by \a right
+///
+////////////////////////////////////////////////////////////
+[[nodiscard]] constexpr Angle operator /(Angle left, Int64 right);
+
+////////////////////////////////////////////////////////////
+/// \relates Angle
+/// \brief Overload of binary /= operator to scale/assign an angle value
+///
+/// \param left  Left operand (an angle)
+/// \param right Right operand (a number)
+///
+/// \return \a left divided by \a right
+///
+////////////////////////////////////////////////////////////
+constexpr Angle& operator /=(Angle& left, float right);
+
+////////////////////////////////////////////////////////////
+/// \relates Angle
+/// \brief Overload of binary /= operator to scale/assign an angle value
+///
+/// \param left  Left operand (an angle)
+/// \param right Right operand (a number)
+///
+/// \return \a left divided by \a right
+///
+////////////////////////////////////////////////////////////
+constexpr Angle& operator /=(Angle& left, Int64 right);
+
+////////////////////////////////////////////////////////////
+/// \relates Angle
+/// \brief Overload of binary / operator to compute the ratio of two angle values
+///
+/// \param left  Left operand (an angle)
+/// \param right Right operand (an angle)
+///
+/// \return \a left divided by \a right
+///
+////////////////////////////////////////////////////////////
+[[nodiscard]] constexpr float operator /(Angle left, Angle right);
+
+////////////////////////////////////////////////////////////
+/// \relates Angle
+/// \brief Overload of binary % operator to compute remainder of an angle value
+///
+/// \param left  Left operand (an angle)
+/// \param right Right operand (an angle)
+///
+/// \return \a left modulo \a right
+///
+////////////////////////////////////////////////////////////
+[[nodiscard]] constexpr Angle operator %(Angle left, Angle right);
+
+////////////////////////////////////////////////////////////
+/// \relates Angle
+/// \brief Overload of binary %= operator to compute/assign remainder of an angle value
+///
+/// \param left  Left operand (an angle)
+/// \param right Right operand (an angle)
+///
+/// \return \a left modulo \a right
+///
+////////////////////////////////////////////////////////////
+constexpr Angle& operator %=(Angle& left, Angle right);
+
+////////////////////////////////////////////////////////////
+/// \relates Angle
+/// \brief User defined literal for angles in degrees
+///
+/// \param angle Angle in degrees
+///
+/// \return \a Angle
+///
+////////////////////////////////////////////////////////////
+[[nodiscard]] constexpr Angle operator "" _deg(long double angle);
```

Comment:
The compiler didn't like me using `float` as the parameter. It might be a language limitation. I'm not totally sure. @vittorioromeo may know more about how this works.

Reply:
https://docs.microsoft.com/en-us/cpp/cpp/user-defined-literals-cpp?view=msvc-170#user-defined-literal-operator-signatures

Looks like using `long double` is required here since a `float` parameter doesn't match one of the standard-approved user defined literal signatures.